internal/scripts/deploy-dotcom.ts

```diff
@@ -387,9 +387,9 @@ async function coalesceWithPreviousAssets(assetsDir: string) {
 
 	// Always include the assets from the directly previous build, but also if there
 	// have been more deploys in the last six months, include those too.
-	const sixMonths = 1000 * 60 * 60 * 24 * 30 * 6
+	const oneMonth = 1000 * 60 * 60 * 24 * 30
```

Comment:
can't we filter on count _and_ time? if the 15,000 limit is one of the factors to consider